superset-frontend/src/welcome/App.jsx

```diff
@@ -50,26 +55,28 @@ const store = createStore(
 );
 
 const App = () => (
-  <Provider store={store}>
-    <Router>
-      <Menu data={menu} />
-      <Switch>
-        <Route path="/superset/welcome/">
-          <Welcome user={user} />
-        </Route>
-        <Route path="/dashboard/list/">
-          <DashboardList user={user} />
-        </Route>
-        <Route path="/chart/list/">
-          <ChartList user={user} />
-        </Route>
-        <Route path="/tablemodelview/list/">
-          <DatasetList user={user} />
-        </Route>
-      </Switch>
-      <ToastPresenter />
-    </Router>
-  </Provider>
+  <ThemeProvider theme={supersetTheme}>
+    <Provider store={store}>
```

Comment:
Hmmm.... might it make sense to swap the layering of <ThemeProvider> and <Provider> so that data from <Provider> can be used to drive the theme?

Reply:
that makes sense!